### ISSUE TITLE
Quarkus CLI "dev" command is not working with arguments

### ISSUE THREAD

**Issue by ia3andy** [quarkusio/quarkus#15887]
## Describe the bug
```shell
qs dev -Dsuspend=true
```

### Expected behavior
Should suspend

### Actual behavior
Unknown option: '-Dsuspend=true'

## To Reproduce

```shell
$ jbang app install qs@quarkusio
$ qs create
$ cd code-with-quarkus
$ qs dev -Dsuspend=true

Unknown option: '-Dsuspend=true'
```

**Comment by aadityasinha-dotcom** [comment]
Can I work on this issue?

**Comment by maxandersen** [comment]
@aadityasinha-dotcom feel free to submit pr for this. Basically just need to add a key/value parameter to the command.

**Comment by ebullient** [comment]
Resolved (or changed/superseded) by #16407 and follow-on items (which add debug/suspend parameters to the dev command AND also allow specifying additional parameters
